relm-core/examples/main.rs

```diff
@@ -60,6 +67,7 @@ enum Msg {
     Quit,
 }
 
+// This is the model, and contains some state
```

Comment:
Missing dot (`.`) and we could add that the model will be updated in the `update` function.